scala-stream-collector/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM java:8-alpine
+
+ENV SCALA_COLLECTOR=20170708 \
+BUILD_PACKAGES='bash curl-dev git curl ca-certificates gettext'
+
+WORKDIR /usr/local/scalacollector
+
+RUN apk update && apk upgrade \
+&& apk add --update $BUILD_PACKAGES \
+&& rm -rf /var/cache/apk/* \
+&& wget -q http://dl.bintray.com/snowplow/snowplow-generic/snowplow_scala_stream_collector_0.9.0.zip \
+&& unzip snowplow_scala_stream_collector_0.9.0.zip \
+&& rm snowplow_scala_stream_collector_0.9.0.zip \
+&& mv snowplow-stream-collector-0.9.0 snowplow-stream-collector
```

Comment:
yep,  good suggestions.  Do not hesitate to ask for anything else, this is very young code and needs some direction!